### ISSUE TITLE
Vidcoder Crash On Launch

### ISSUE THREAD

**Issue by ChiggaYuan** [RandomEngy/VidCoder#318]
Whenever I launch Vidcoder the interface appears but I can't click anything and it just tells me that the application stopped working. Does anyone know how to fix this?

**Comment by RandomEngy** [comment]
What version are you using? What happens when you rename %appdata%\VidCoder\VidCoder.sqlite ?

**Comment by ChiggaYuan** [comment]
I can't find the file in the Vidcoder folder. I have 2.63 x64 installed right now.

**Comment by RandomEngy** [comment]
What happens when you type Win+R and put in %appdata%\VidCoder ? Do you not see any files?

**Comment by ChiggaYuan** [comment]
Yes, I see the file you were talking. Whenever I rename the file a new one appears that's called VidCoder.sqlite

**Comment by RandomEngy** [comment]
But that doesn't help the crash? Can you try the beta version? If that crashes try the Handbrake nightly. That will tell me if the issue is with the Handbrake core or not.

**Comment by ChiggaYuan** [comment]
I've tried the beta version already and it crashes on launch as well. I can open HandBrake tho.

**Comment by RandomEngy** [comment]
Curses. Could you try going to Event Viewer -> Application and seeing if any error showed up for the crash?

**Comment by ChiggaYuan** [comment]
How do I get there?

**Comment by RandomEngy** [comment]
Start, then type "event viewer"

**Comment by ChiggaYuan** [comment]
I got in the Event viewer and I clicked Applications and Services, what do I do after?

**Comment by RandomEngy** [comment]
Click on "Application" and look for errors related to VidCoder.
